src/util/audiusSdk.js

```diff
@@ -0,0 +1,11 @@
+import { sdk } from '@audius/sdk'
+
+let audiusSdk = null
+
+const initAudiusSdk = () => {
+  audiusSdk = sdk({ appName: 'Audius Bedtime Client' })
```

Comment:
i think we want to move away from using bedtime publicly and this will show on the protocol dashboard, let's use

'Audius Embed Player'